### ISSUE TITLE
Plans for next update?

### ISSUE THREAD

**Issue by TheRowdy** [Dijji/FileMeta#46]
Are you planning/working on a new update? If so, when should we expect the new release, and what will it add primarily?

**Comment by Dijji** [comment]
No, I have no updates planned at the moment. If I could figure out how to get Explorer to show folder meta data, I would love to do that, but I don’t have any other features in mind at the moment.

Do you have any suggestions?

Dijji

**Comment by Hept0p** [comment]
#35

**Comment by Hept0p** [comment]
#35
